src/convertToObject.js

```diff
@@ -7,6 +7,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((entry) => entry.trim())
+    .filter((entry) => entry)
+    .reduce((cssProperties, entry) => {
```

Comment:
The indentation of chained methods is not consistent. To improve readability, each chained method should be on a new line with the same indentation level.